tools/cdk-build-tools/lib/os.ts

```diff
@@ -27,6 +34,10 @@ export async function shell(command: string[], timers?: Timers): Promise<string>
       stdout.push(chunk);
     });
 
+    child.stderr!.on('data', chunk => {
+      process.stderr.write(makeRed(chunk.toString()));
```

Comment:
`chunk` *might* include an incomplete multi-byte unicode point, and this would effectively corrupt it 😕